Formula/d/dotlocalctl.rb

```diff
@@ -0,0 +1,22 @@
+class Dotlocalctl < Formula
+  desc "Backend CLI for DotLocal"
+  homepage "https://degreat.co.uk/dotlocal/cli"
+  url "https://degreat.co.uk/packages/dotlocalctl-0.0.2.tar.gz"
```

Comment:
is this open source project? if so, where is the code?

Reply:
No, it's not open source. Is it required to be open source?

Reply:
Yes, https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae

Reply:
To be clear, you can still make your software available through Homebrew regardless of its license by hosting the formula in your own [tap](https://docs.brew.sh/Taps), basically a third-party repository that the `brew` tool can install software from.

But to consider software for inclusion in _this_ repo (homebrew-core), the software must be open-source, per the link that Sean shared.